app/src/sharedTest/java/org/oppia/android/app/options/ReadingTextSizeFragmentTest.kt

```diff
@@ -82,13 +75,23 @@ import org.robolectric.annotation.LooperMode
 import javax.inject.Inject
 import javax.inject.Singleton
 
-private const val SMALL_TEXT_SIZE = 0
-private const val MEDIUM_TEXT_SIZE = 5
-private const val LARGE_TEXT_SIZE = 10
+private const val SMALL_TEXT_SIZE_SCALE = 0.8f
```

Comment:
Add following comments:
1. These duplicates are intentional to make sure that that the production values are not changed by mistake.
2. TODO #1815: Remove these duplicate values once Screenshot tests are implemented.

@ARJUPTA PTAL

Reply:
@rt4914 LGTM

Reply:
Is this TODO format is correct?

Reply:
Format can be confirmed from code. I just mentioned it as points.

Reply:
@ARJUPTA Format these comments correctly.
```
// These duplicates are intentional to make sure that that the production values are not changed 
// by mistake.
// TODO(#59): 1815: Remove these duplicate values once Screenshot tests are implemented. 
```

The above comment was for understanding not in comments format.